src/hooks/login/useExtensionLogin.ts

```diff
@@ -45,7 +45,7 @@ export const useExtensionLogin = ({
       }
 
       const callbackUrl: string = encodeURIComponent(
-        `${window.location.origin}${callbackRoute}`
+        `${window.location.origin}${callbackRoute ?? window.location.href}`
```

Comment:
we need to get the pathname here, because there's a window.location.origin prefix and this will break the URL